PowerEditor/src/NppCommands.cpp

```diff
@@ -2273,17 +2273,16 @@ void Notepad_plus::command(int id)
 
 		case IDM_VIEW_REDUCETABBAR:

```

Comment:
should also handle padding
```c++
TabCtrl_SetPadding(_mainDocTab.getHSelf(), _mainDocTab.dpiManager().scale(TabBarPlus::drawTabCloseButton() ? 10 : 6), 0);
```
values from `void TabBarPlus::doOwnerDrawTab(TabBarPlus* tbpObj)`